### ISSUE TITLE
repair few mechanics in game

### ISSUE THREAD

**Issue by bloodyplayer** [AndroidModLoader/JPatch#24]
As in GTA VC mobile we can leave a car door open while exiting and Tommy can dodge jump when a car is approaching him fast but in GTA SA both r broken
The game can't sense the danger that that cj is about to get run over by a vehicle approaching I guess that.
Can the classic weapon don't render thru glass can be fixed?
Can these be fixed in future

**Comment by bloodyplayer** [comment]
Rarely horse gambling machines make wrong sounds and after the voice of the police radio is played radio tuning is keep on playing on background till i play the game

**Comment by bloodyplayer** [comment]
When going thru tear gas smoke cj must cough alike NPCs and drop some health slowly.
It would be cool if cj scream after saying a line when falling like niko or Franklin in HD UNIVERSE

**Comment by RusJJ** [comment]
> When going thru tear gas smoke cj must cough alike NPCs and drop some health slowly.
Tear gas is incomplete and not available without cheats.

> It would be cool if cj scream after saying a line when falling like niko or Franklin in HD UNIVERSE
CJ already does that.

**Comment by bloodyplayer** [comment]
> 
> > It would be cool if cj scream after saying a line when falling like niko or Franklin in HD UNIVERSE
> 
> CJ already does that.
Yes but he shouts one time and screams for 4 seconds only.. i rest of the time he falls silently.

**Comment by RusJJ** [comment]
Weapons rendering through the glass been fixed lately
Dodging the vehicle? It was removed
